Extension/package.nls.json

```diff
@@ -17,6 +17,64 @@
     "c_cpp.command.rescanWorkspace.title": "Rescan Workspace",
     "c_cpp.command.vcpkgClipboardInstallSuggested.title": "Copy vcpkg install command to clipboard",
     "c_cpp.command.vcpkgOnlineHelpSuggested.title": "Visit the vcpkg help page",
+    "c_cpp.configuration.formatting.description": "Select Formatting Engine",
```

Comment:
I think you'll want to specify more here.  There was an existing description for this that was removed, that would be a good starting point.